router/group.go

```diff
@@ -104,18 +112,74 @@ func (h *Handlers) DeleteGroup(c echo.Context) error {
 }
 
 func (h *Handlers) GetMembers(c echo.Context) error {
-	return c.NoContent(http.StatusOK)
-	// TODO: Implement
+	groupID, err := uuid.Parse(c.Param("groupID"))
+	if err != nil {
+		return badRequest(err)
+	}
+	if groupID == uuid.Nil {
+		return badRequest(err)
+	}
+
+	ctx := context.Background()
+	members, err := h.Repository.GetMembers(ctx, groupID)
+	if err != nil {
+		return internalServerError(err)
+	}
+
+	var res []uuid.UUID
+	for _, member := range members {
+		res = append(res, member.ID)
+	}
+
+	return c.JSON(http.StatusOK, &MemberResponse{res})
 }
 
 func (h *Handlers) PostMember(c echo.Context) error {
-	return c.NoContent(http.StatusOK)
-	// TODO: Implement
+	var member Member
+	if err := c.Bind(&member); err != nil {
+		return badRequest(err)
+	}
+
+	groupID, err := uuid.Parse(c.Param("groupID"))
+	if err != nil {
+		return badRequest(err)
+	}
+	if groupID == uuid.Nil {
+		return badRequest(err)
+	}
+
+	ctx := context.Background()
+	created, err := h.Repository.CreateMember(ctx, groupID, member.ID)
+	if err != nil {
+		return internalServerError(err)
+	}
```

Comment:
```suggestion
	if err != nil {
		if ent.IsConstraintError(err) {
			return badRequest(err)
		}
		return internalServerError(err)
	}
```
そのUUIDを持つUser/Groupが存在しないときは`InternalServerError`よりも`BadRequest`を返したいので、このように`ent`の`IsConstraintError`を利用して`BadRequest`を返すように修正してもらいたいです。
合わせてテストも変更するかも？見てないけど必要あったら修正しておいてもらえると:pray: